cumulus/parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -230,6 +230,9 @@ match_types! {
 		MultiLocation { parents: 1, interior: Here } |
 		MultiLocation { parents: 1, interior: X1(_) }
 	};
+	pub type TreasuryPallet: impl Contains<MultiLocation> = {
+		MultiLocation { parents: 1, interior: X1(PalletInstance(18)) }
+	};
```

Comment:
Maybe a comment to document how this is accessed? As in, is it via the `Treasurer` track or some autonomous logic?

Reply:
not sure I understand. this is just a `Contains` type with treasury pallet location